src/index.js

```diff
@@ -21,5 +22,6 @@ if (winston) {provisionWinston(winston, Transport);}
 
 module.exports = {
```

Comment:
```module.exports = {
bunyanStream: buildBunyanStream,
<whatever this was> : provisionWinston,
...Logger};```

That way the `Logger.bunyanStream` will continue working :)

Reply:
`Logger.bunyanStream` won't work unless you do
`const Logger = require('r7insight_node);`
which would mean that you would have to do
`new Logger.Logger()` to access the logger, which makes no sense so we should only use named imports, e.g. `const { buildBunyanStream } = require('r7insight_node')`

Also this is confusing since `BunyanStream` is a class within the package, and exporting a bunyanStream will confuse people thinking it's a class, not a builder/factory.

Reply:
Also for the first q, the only exports were the logger (containing winston and bunyan integration), and a load of events:
```
export {
    Logger as default,
    errorEvent,
    logEvent,
    connectedEvent,
    disconnectedEvent,
    timeoutEvent,
    drainWritableEvent,
    finishWritableEvent,
    pipeWritableEvent,
    unpipeWritableEvent,
    bufferDrainEvent
};
```
but all of these are documented in the README, and I personally don't see the point of exporting them. Rather than the user having to do
```
const { bufferDrainEvent } = require('r7insight_node');
logger.once(bufferDrainEvent, () => {
```
they can just do 
```
logger.once('buffer drain', () => {
```
but if you think they would be useful, sure I can export them